### PR TITLE
Allow upgrading from ownCloud 10.11

### LINKED ISSUE #34049: Migrating from ownCloud 10.10 and 10.11
**Is your feature request related to a problem? Please describe.**
I have a up-to-date ownCloud server (version 10.10) and I want to migrate to nextCloud but I get the message "Exception: Updates between multiple major versions and downgrades are unsupported.". I have tried version 24.0.5 (last) and 20.0.14 of nextCloud with same results.
In "version.php" file OC_VersionCanBeUpgradedFrom indicate migrations can only occurs from owncloud 10.5 .

**Describe the solution you'd like**
Migrating from ownCloud 10.10 should be possible.

### PR DESCRIPTION
**Pull request by PVince81**
* Resolves: https://github.com/nextcloud/server/issues/34049

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

Tested here https://github.com/nextcloud/server/issues/34049#issuecomment-1329749268

### CODE REVIEW

**Comment by PVince81** [comment]
docs PR here: https://github.com/nextcloud/documentation/pull/9424

**Review by mejo- — APPROVED** [review]

File: version.php [hunk 1]
```diff
@@ -42,6 +42,7 @@
 	],
 	'owncloud' => [
 		'10.5' => true,
+		'10.11' => true,
```
Comment (szaimen) [inline version.php:45]: Should be maybe overwrite 10.5 or do we want to allow to upgrade from both?

**Review by szaimen — APPROVED** [review]
LGTM but didnt test

File: version.php (hunk 1 above)
Comment (PVince81) [reply version.php:45]: yeah, I hesitated to do that. let's overwrite

**Review by come-nc — APPROVED** [review]

**Pull request merged**